src/slack/slack.js

```diff
@@ -360,39 +360,32 @@ export class Slack {
 
   async getComments(ts, channel) {
     const fetchComments = async (cursor) => {
-      // TODO: catch error (test by asAdmin: false)
-      let messages, nextCursor
-
-      try {
-        ({messages, response_metadata: {next_cursor: nextCursor} = {}} = await this.boltApp.client.conversations.replies({channel, ts, cursor}))
-      } catch (err) {
-        logger.error(`Failed to fetch comments: ${err}`)
-      }
-
+      // error is caught later
+      const {messages, response_metadata: {nextCursor} = {}} = await this.boltApp.client.conversations.replies({channel, ts, cursor})
```

Comment:
Let do:

`const {messages, response_metadata: {next_cursor: nextCursor} = {}}`

Otherwise it gives eslint warning: **_Identifier 'next_cursor' is not in camel case.eslint_**